spec.md

```diff
@@ -1737,6 +1737,12 @@ message ListSnapshotsRequest {
   // and will not block if the snapshot is being processed after
   // it is cut.
   string snapshot_id = 4;
+
+  // Secrets necessary for listing snapshots.
+  // This field is OPTIONAL. It can be used to pass secrets
+  // required to access some subset of the snapshots in the
```

Comment:
What do you mean by "some subset of the snapshots"?  How is a subset defined?

Reply:
How about `... access the snapshots...`?

Reply:
How about:

```
  // Secrets required by plugin to complete ListSnapshot request.
  // This field is OPTIONAL. Refer to the `Secrets Requirements`
  // section on how to use this field.
```